superset/extensions/ssh.py

```diff
@@ -63,8 +63,8 @@ def create_tunnel(
         elif ssh_tunnel.private_key:
             private_key_file = StringIO(ssh_tunnel.private_key)
             private_key = RSAKey.from_private_key(private_key_file)
-            params["private_key"] = private_key
-            params["private_key_password"] = ssh_tunnel.private_key_password
+            params["ssh_pkey"] = private_key
+            params["ssh_private_key_password"] = ssh_tunnel.private_key_password
```

Comment:
Oh! I was reading the docs here: https://sshtunnel.readthedocs.io/en/latest/ thanks!
Also, not a blocker for these changes, but probably worth mentioning we should make sure that when updating a tunnel, if we send a `password` or `private_key/private_key_password`, we remove the other login method from the tunnel (the UI is doing it already), otherwise, it will get into the `if` all the time if both are present, even if `password` is not the latest login method of the tunnel.

Reply:
Let's cut a ticket for this as well on our end